doc/proxy-clients.rst

```diff
@@ -202,14 +202,14 @@ The recommended usage of the proxy client is to create an instance of
 Implementation Notes
 --------------------
 
-Each client is an implementation of :source:`ProxyClientInterface <src/ProxyClient/ProxyClientInterface.php>`.
-All other interfaces, ``PurgeInterface``, ``RefreshInterface`` and ``BanInterface``
-extend this ``ProxyClientInterface``. So each client implements at least one of
+Each client is an implementation of :source:`ProxyClient <src/ProxyClient/ProxyClient.php>`.
+All other interfaces, ``Purge``, ``Refresh`` and ``Ban``
+extend this ``ProxyClient``. So each client implements at least one of
 the three :ref:`invalidation methods <invalidation methods>` depending on the
```

Comment:
not related to your change, but TagCapable is now its own interface, so there are four things.